model/resource/faas.yaml

```diff
@@ -1,6 +1,7 @@
 groups:
   - id: faas_resource
     type: resource
+    name: faas
```

Comment:
Do we need `id` to be different from `name` in this case? I was wondering if we could simplify and keep just the `name`.